enterprise/frontend/src/embedding-sdk/hooks/public/use-question-search.ts

```diff
@@ -21,6 +20,6 @@ export const useQuestionSearch = (searchQuery?: string) => {
 
   return useSearchListQuery({
     query,
-    enabled: isLoggedIn && isInitialized,
+    enabled: isLoggedIn,
```

Comment:
Initialization (in theory) is always a prerequisite of logging in, so we should just use this